### ISSUE TITLE
Italic a looks like italic o and is a departure from normal a

### ISSUE THREAD

**Issue by PatrickF1** [JetBrains/JetBrainsMono#645]
## Problem
Italic 'a' and italic 'o' look quite alike. Furthermore, it's a bit surprising that the italic 'a' is a not just a slanted form of the regular a.
![Screenshot 2023-10-02 at 11 46 13 AM](https://github.com/JetBrains/JetBrainsMono/assets/1967248/88ebfa5e-87bc-40c3-b091-ab948d8b3686)

## Proposal
Make the italic 'a' simply a slanted version of the regular 'a'.

Just my two cents. I think this is my biggest issue with the JetBrains Mono font. Besides this, almost everything is perfect. Thank you for making such a great font!

**Comment by toothbrush7777777** [comment]
I disagree. I think that the changed letter forms make the italicised text much easier to read. Besides, italic monospace text is usually used for comments or keywords. If you want to be picky, `D` and `O` also look alike.

**Comment by yvvt0379** [comment]
Yeah, in former versions, the italic a didn't have tail, which looked even more closer to o. The tail exists to distinguish from o.

**Comment by PatrickF1** [comment]
Oh, so the italic a had the same form as the normal a and was just slanted and was even harder to distinguish from o?

**Comment by yvvt0379** [comment]
No, it was a single-story a without tail.

**Comment by PatrickF1** [comment]
Oh I see what you mean. It was this shape before but without the tail. They
added a tail recognizing it wasn’t distinguishable without it and so that’s
why they consider the issue solved.
